appveyor.yml

```diff
@@ -0,0 +1,23 @@
+version: "{build}"
+
+clone_folder: c:\gopath\src\github.com\docker\cli
+
+environment:
+  GOPATH: c:\gopath
+  GOVERSION: 1.10
+  DEPVERSION: v0.4.1
+
+install:
+  - rmdir c:\go /s /q
+  - appveyor DownloadFile https://storage.googleapis.com/golang/go%GOVERSION%.windows-amd64.msi
```

Comment:
Shouldn't that be prefixed with ```appveyor-retry``` just to be on the safe side ?

Reply:
@mat007 noted, thanks :stuck_out_tongue: We never had any problem so far with that, so I'm fine merging like this. But if somehow it turns out we need to be more on the safe side, we'll use `appveyor-retry` then :wink: